curriculum/challenges/english/07-scientific-computing-with-python/learn-lambda-functions-by-building-an-expense-tracker/65824561f3478e5371a33ae5.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: 65824561f3478e5371a33ae5
-title: Step 23
+title: Step 24
 challengeType: 20
-dashedName: step-23
+dashedName: step-24
 ---
 
 # --description--
```

Comment:
It is better to introduce `main()` in detail in the [first step](http://localhost:8000/learn/scientific-computing-with-python/learn-how-to-work-with-numbers-and-strings-by-implementing-the-luhn-algorithm/step-1) of Luhn Algorithm project to avoid explaining in later in the curriculum. There `main()` is not properly introduced.

Reply:
What should we say about it?

Reply:
#54733 should resolve this.